cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java

```diff
@@ -28,20 +28,19 @@
 
   static final String CONFIG_NAME = "iotdb-cluster.properties";
 
-  private String clusterRpcIp = "127.0.0.1";
+  private String internalIp = "127.0.0.1";
   private int internalMetaPort = 9003;
   private int internalDataPort = 40010;
-  private int clusterRpcPort = 55560;
+  private int clusterRpcPort = 6667;
 
-  /** each one is a "<IP | domain name>:<meta port>:<data port>:<client port></>" string tuple */
+  /** each one is a {internalIp | domain name}:{meta port} string tuple */
   private List<String> seedNodeUrls =
-      Arrays.asList(
-          "127.0.0.1:9003:40010:55560", "127.0.0.1:9005:40012:55561", "127.0.0.1:9007:40014:55562");
+      Arrays.asList(String.format("%s:%d", internalIp, internalMetaPort));
```

Comment:
why not use seed_nodes?

Reply:
Sorry, I don't know what it means. This is to initialize the seed_ nodes

Reply:
OK, just leave this review away